public/app/plugins/datasource/testdata/query_ctrl.ts

```diff
@@ -30,7 +30,7 @@ export class TestDataQueryCtrl extends QueryCtrl {
     this.scenarioList = [];
     this.newPointTime = dateTime();
     this.selectedPoint = { text: 'Select point', value: null };
-    this.target.squareWave = _.defaults(this.target.squareWave || {}, defaultSquare);
+    this.target.pulseWave = _.defaults(this.target.pulseWave || {}, defaultPulse);
```

Comment:
can you move this to the `scenarioChanged() {` section?

This will add the default parameters for every request, not just pulse ones